smoke-tests/fake-backend/build.gradle

```diff
@@ -30,10 +30,8 @@ dependencies {
 }
 
 jib {
-  to {
-    image = "open-telemetry-docker-dev.bintray.io/java/smoke-fake-backend"
-  }
-  from {
-    image = "gcr.io/distroless/java-debian10:11"
-  }
+  container.creationTime = 'USE_CURRENT_TIMESTAMP'
+  from.image = "gcr.io/distroless/java-debian10:11"
+  to.image = "open-telemetry-docker-dev.bintray.io/java/smoke-fake-backend"
+  to.tags = [new Date().format("yyyyMMdd.HHmmSS")]
```

Comment:
Will that add `latest` tag as well?

Reply:
According to specification, `latest` tag is pointing at the latest specifically untagged image (which is `to.image`). This is what my docker images shows me:
```
REPOSITORY                                                              TAG                     IMAGE ID            CREATED             SIZE
open-telemetry-docker-dev.bintray.io/java/smoke-springboot-jdk11        20201104.1241567        c9cef0f61231        14 seconds ago      147MB
open-telemetry-docker-dev.bintray.io/java/smoke-springboot-jdk11        latest                  c9cef0f61231        14 seconds ago      147MB
open-telemetry-docker-dev.bintray.io/java/smoke-grpc-jdk11              20201104.1227972        bdff883aafb7        14 minutes ago      149MB
open-telemetry-docker-dev.bintray.io/java/smoke-grpc-jdk11              latest                  bdff883aafb7        14 minutes ago      149MB
open-telemetry-docker-dev.bintray.io/java/smoke-fake-backend            20201104.1225631        de3c1e136bb9        15 minutes ago      231MB
open-telemetry-docker-dev.bintray.io/java/smoke-fake-backend            latest                  de3c1e136bb9        15 minutes ago      231MB
open-telemetry-docker-dev.bintray.io/java/smoke-springboot-jdk11        20201104.1222643        ce4f7244e018        19 minutes ago      147MB
```